### ISSUE TITLE
Proposal: Automatic Package Compilation

### ISSUE THREAD

**Issue by brunoalano** [Miserlou/lambda-packages#37]
In some sense, the package compilation for AWS Lambda it's simple, following some basic steps, described in the README for example, where you should install the package dependencies and then compile, zip, and PR.

It's possible to make this process automatic using a tool like Travis? Otherwise, I can make available an instance on EC2 running the correct AMI to that purposes, where should have some listener (Flask for example) that runs the process.

**Comment by Miserlou** [comment]
Yeah, I've been thinking about this for a long time, but haven't had the bandwidth to really address it properly.

I think another option to look into is simply building on top of something like https://github.com/lambci/docker-lambda - we might be able to skip the EC2 instances entirely (although we loose a little bit of assurance I'm not too worried about it if the Docker images are doing their job).

**Comment by brunoalano** [comment]
See: https://www.rickpeyton.com/posts/docker-plus-amazonlinux-to-build-python-aws-lambda-package.html

**Comment by Miserlou** [comment]
Yeah, pretty much that. The goal would be reproducability of the packages. Are we missing any other major packages that aren't in l-p or manylinux?
